docs/user/cli/geotools.rst

```diff
@@ -0,0 +1,102 @@
+.. _gt_tools:
+
+GeoTools Command-Line Tools
+===========================
+
+The GeoMesa GeoTools distribution includes a set of command-line tools that can work with most non-GeoMesa
+data store implementations to provide feature management, ingest and export. This allows the use of GeoMesa
+converters and output encodings to be used with non-GeoMesa data stores.
+
+Installation
+------------
+
+GeoMesa GeoTools artifacts are available for download or can be built from source.
+The easiest way to get started is to download the most recent binary version
+(|release|) from `GitHub`__.
+
+__ https://github.com/locationtech/geomesa/releases
+
+Extract it somewhere convenient:
+
+.. code-block:: bash
+
+    # download and unpackage the most recent distribution:
+    $ wget "https://github.com/locationtech/geomesa/releases/download/geomesa_2.11-$VERSION/geomesa-gt_2.11-$VERSION-bin.tar.gz"
+    $ tar xvf geomesa-gt_2.11-$VERSION-bin.tar.gz
+    $ cd geomesa-gt_2.11-$VERSION
+    $ ls
+    bin/  conf/  dist/  docs/  examples/  lib/  LICENSE.txt  logs/
+
+Alternatively, it may be built from source. For more information refer to :ref:`building_from_source`
+in the developer manual, or to the ``README.md`` file in the the source distribution. If you have built from
+source, the distribution is created in the ``target`` directory of ``geomesa-gt/geomesa-gt-dist``.
+
+Setting up the Command Line Tools
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once installed, the command line tools can be invoked by running the script ``geomesa-gt`` located in the binary
+distribution under ``geomesa-gt_2.11-$VERSION/bin/``.
+
+The tools ship with some default GeoTools data stores, such as support for Postgis and shapefiles. For other stores,
+you will need to copy the appropriate JARs into the tools ``lib`` folder.
+
+.. note::
+
+    You can configure environment variables and classpath settings in ``geomesa-gt_2.11-$VERSION/conf/geomesa-env.sh``.
+
+.. note::
+
+    ``geomesa-gt`` will load JARs from the ``$GEOMESA_EXTRA_CLASSPATHS`` environment variable
+    into the class path. Use the ``geomesa-gt classpath`` command in order to see what JARs are being used.
+
+
+Due to licensing restrictions, dependencies for shape file support must be separately installed. Do this with the
+following commands:
+
+.. code-block:: bash
+
+    $ bin/install-jai.sh
+    $ bin/install-jline.sh
+
+Run ``geomesa-gt`` without arguments to confirm that the tools work.
+
+.. code::
+
+    $ bin/geomesa-gt
+    INFO  Usage: geomesa-gt [command] [command options]
+      Commands:
+      ...
+
+Setting up Distributed Processing
```

Comment:
Does MR ingest, etc work with the various GeoTools datastores?  Seems like this might deserve a note/warning;).

Reply:
Yeah it does, what note/warning were you thinking of? I thought this section was basically the note/warning.